frontend/src/components/pages/home.vue

```diff
@@ -1,12 +1,20 @@
 <template>
-  <v-container fluid primary align-center justify-center fill-height class="raise">
+  <v-container fluid align-center justify-center fill-height class="raise">
 
         <v-row>
             <v-col cols=12>
-                <p style="font-size: 100pt">BC 💖 {{$tc('Data')}}</p>
+                <v-row align="center" justify="center">
+                    <v-card tile class="mx-3 pa-6" height="200px" width="200px" ><p style="font-size: 50pt">BC</p></v-card>
```

Comment:
This works for now, might want to consider adding the photo and text areas of the card though based on the design

Reply:
I think this is best left for later, we have no idea what's going in the cards yet